src/fontra/views/editor/editor.js

```diff
@@ -1501,6 +1525,13 @@ export class EditorController {
   }
 
   canPaste() {
+    const glyphLocked =
+      this.sceneModel.getSelectedPositionedGlyph()?.varGlyph.glyph.customData?.[
```

Comment:
Try to paste onto a non-existing glyph: `Cannot read properties of null (reading 'glyph')`

This fixes it:
```suggestion
      this.sceneModel.getSelectedPositionedGlyph()?.varGlyph?.glyph.customData?.[
```

Reply:
Fixed it in `isSelectedGlyphLocked`